src/app/app.module.ts

```diff
@@ -3,23 +3,31 @@ import { BrowserModule } from '@angular/platform-browser';
 
 import { AppRoutingModule } from './app-routing.module';
 import { AppComponent } from './app.component';
-import { DiaryCardComponent } from './components/diary-card/diary-card.component';
+import { DiaryCardComponent } from './module-1/components/diary-card/diary-card.component';
```

Comment:
In the screenshot `module-1` is taken as a sample module to give you an idea. You should use relevant name to your module.

Reply:
resoleved